content/en/Integrations/webhooks.md

```diff
@@ -0,0 +1,92 @@
+
+## Configure Webhooks in the UI
+
+Let’s configure webhooks in the Cobalt app.
+
+### Create a Webhook
+
+To create a webhook:
+
+1. In the Cobalt app, navigate to the **Integrations** page, and then select **Webhooks** under **Native Integrations**.
+1. On the **Webhooks** page, select **Create Webhook**.
+1. In the overlay that appears, specify the following:
+   - **Webhook Name**
+   - **Webhook URL**: URL to which Cobalt sends HTTP POST requests for pentest [events](#webhook-events).
+     - Use a unique name and URL for each webhook you create.
+   - **Authentication Token**: Your [token](#before-you-start) required to authorize API requests that Cobalt sends to your URL.
+1. When ready, select **Save**.
+1. We send a test API request to the specified URL to validate your webhook. The webhook becomes active once the validation is complete.
+   - If the validation fails, we’ll disable your webhook within 24 hours. See [Troubleshooting](#troubleshooting) for more information.
+
+![Create a webhook in the Cobalt UI](/integrations/CreateWebhook.png "Create a webhook in the Cobalt UI")
+
+### Manage Webhooks
+
+Now you can manage the webhooks that you [created](#create-a-webhook).
+
+- To make a webhook **inactive**, use the toggle under **Active**.
+- To **edit** a webhook, select the three-dot icon under **Actions**, and then select **Edit Webhook**. Update webhook parameters in the overlay that appears.
+- To **delete** a webhook, select the three-dot icon under **Actions**, and then select **Delete Webhook**.
+  - You can make a webhook inactive without deleting it.
+
+## Troubleshooting
+
+You can check the status of your webhooks on the **Webhooks** page. The following icons indicate that there is a problem with your webhook:
+
+- **Yellow warning**: The webhook stopped responding to events at the specified time. We’ll disable the webhook after 48 hours of failed attempts.
+- **Red error**: The webhook stopped responding, and we disabled it after 48 hours of failed attempts.
+
+Find solutions for common troubleshooting problems in the table below:
+
+| Problem | Solution |
+|---|---|
+| You can’t create a webhook because the name or URL already exists | Enter a unique name and URL for your webhook |
+| <li>You can’t create a webhook because the validation fails</li><li>You don’t get webhook notifications to your URL</li><li>You can't enable a webhook that we disabled or you made inactive earlier</li> | We failed to validate your webhook. Check the following:<li>Your authentication token is valid. Refresh the token or generate a new one if needed.</li><li>The URL is valid and can accept requests.</li>|
+| You can't delete a webhook | Try again or contact support |
```

Comment:
One more item. (My bad for not catching this earlier.)

In two other locations ([here](https://github.com/cobalthq/cobalt-product-public-docs/blame/main/content/en/Getting%20started/Assets/asset-description.md#L110) and [here](https://github.com/cobalthq/cobalt-product-public-docs/blame/3a56912d250538dbb872e81685f18802abbba604/content/en/Getting%20started/what-to-expect.md#L58)), we've said "Contact your Customer Success Manager (CSM) or support@cobalt.io."

Maybe it's time to set up a "partial" with that statement?

Reply:
@mjang-cobalt Here is a minor PR for that: https://github.com/cobalthq/cobalt-product-public-docs/pull/173